Sources/Kommunicate/Classes/KMConversationViewController.swift

```diff
@@ -135,6 +135,7 @@ open class KMConversationViewController: ALKConversationViewController {
         guard let channelId = viewModel.channelKey else { return }
         sendConversationOpenNotification(channelId: String(describing: channelId))
         setupConversationClosedView()
+        delayInterval = KMAppUserDefaultHandler.shared.botMessageDelayInterval/1000
```

Comment:
Are we using it anywhere?

Reply:
We are not using this anywhere. Found a workaround without this . We can close this PR.